x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_panel.scss

```diff
@@ -16,7 +16,7 @@
 
 .lnsLayerPanel__row {
   background: $euiColorLightestShade;
-  padding: $euiSizeS;
+  padding: $euiSizeS 0;
   border-radius: $euiBorderRadius;
 
   // Add margin to the top of the next same panel
```

Comment:
Possible to change `margin-top: $euiSizeS;` to `margin-top: $euiSize;` to better match the provided designs?